ui/raidboss/data/04-sb/dungeon/st_mocianne_hard.js

```diff
@@ -0,0 +1,141 @@
+import Conditions from '../../../../../resources/conditions.js';
+import NetRegexes from '../../../../../resources/netregexes.js';
+import { Responses } from '../../../../../resources/responses.js';
+import ZoneId from '../../../../../resources/zone_id.js';
+
+export default {
+  zoneId: ZoneId.SaintMociannesArboretumHard,
+  timelineFile: 'st_mocianne_hard.txt',
+  timelineTriggers: [
+    {
+      id: 'St Mocianne Hard Quickmire',
+      regex: /Quickmire/,
+      beforeSeconds: 7, // This is approximately when the sewage surge begins.
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Be On A Platform',
+        },
+      },
+    },
+  ],
+  triggers: [
+    {
+      // Trash mob gaze attack
+      id: 'St Mocianne Hard Frond Fatale',
+      netRegex: NetRegexes.startsUsing({ id: '31A4', source: 'Withered Belladonna' }),
+      condition: (data) => data.CanSilence(),
+      response: Responses.interrupt(),
+    },
+    {
+      id: 'St Mocianne Hard Vine Whip',
+      netRegex: NetRegexes.startsUsing({ id: '2E48', source: 'Nullchu' }),
+      condition: Conditions.caresAboutPhysical(),
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'St Mocianne Hard Sludge Bomb',
+      netRegex: NetRegexes.headMarker({ id: '0001' }),
+      condition: Conditions.targetIsYou(),
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Sludge puddle on YOU',
+        },
+      },
+    },
+    {
+      id: 'St Mocianne Hard Fault Warren',
+      netRegex: NetRegexes.headMarker({ id: '003E' }),
+      suppressSeconds: 5,
```

Comment:
Can you leave a comment explaining this one? I think it's something I could see accidentally removing later or not understanding why it's here.

Reply:
Good point!